spec/system/users/edit_spec.rb

```diff
@@ -99,6 +99,16 @@
       expect(page).to have_content "1 error prohibited this Volunteer from being saved:"
       expect(page).to have_text("At least one communication preference must be selected.")
     end
+
+    it "displays notification events selection as enabled if sms notification preference is selected" do
+      check "user_receive_sms_notifications"
+      expect(page).to have_field("toggle-sms-notification-event", type: "checkbox", disabled: false)
+    end
+
+    it "displays notification events selection as disabled if sms notification preference is not selected" do
+      uncheck "user_receive_sms_notifications"
+      expect(page).to have_field("toggle-sms-notification-event", type: "checkbox", disabled: true)
```

Comment:
This test should be passing but seems like the JS is not running in capybara. Any clues on how to do that?